### DISCUSSION TITLE
The first time I used mmaction2 for a demo inference, I was confused.

### DISCUSSION THREAD

**Discussion by nlxxWangYH** [open-mmlab/mmaction2#2582] [Q&A]
The example of using the official demo is as follows
![image](https://github.com/open-mmlab/mmaction2/assets/87682766/ab1bcab5-15d3-4756-afb7-6328aa83351f)
There are some questions of incomprehension
![1688654479941](https://github.com/open-mmlab/mmaction2/assets/87682766/cd653fdb-1b35-41e8-8fa5-2a48ae9bd1b9)
Why the Performing Human Detection for each frame progress is zero?
Why Performing SpatioTemporal Action Detection for each clip progress 27/28?
Whether the "problem" mentioned above is really a problem, and if so, what impact it will have, and how it should be solved.
I know my question is naïve, but as a newbie I am really confused, I hope to get answers from everyone, thank you very much！

**Comment by cir7** [comment]
I guess the progress is flushed by the following warning message, and has no effect on the result.
